pyproject.toml

```diff
@@ -36,7 +36,7 @@ xcode-project = "codemagic.tools:XcodeProject.invoke_cli"
 
 [tool.poetry.dependencies]
 python = "^3.7"
-cryptography = ">= 3.3, != 37.0.0"
+cryptography = ">= 38.0.0"
```

Comment:
Due to changes below we're now using [`PrivateFormat.PKCS12`](https://cryptography.io/en/latest/hazmat/primitives/asymmetric/serialization/#cryptography.hazmat.primitives.serialization.PrivateFormat.PKCS12) from `cryptography`, which was added only in version [38.0.0](https://pypi.org/project/cryptography/38.0.0/). But it should be fine as this version is already close to two years old and installing anything older than 35.0.0 is very questionable anyway.

Reply:
Worth adding a note to changeling, perhaps?

Reply:
Yes, of course! I started to rush a little last night.

Reply:
@artemii-yanushevskyi, I updated changelog in [76dcb58](https://github.com/codemagic-ci-cd/cli-tools/pull/411/commits/76dcb58b9e2e26f863f89422803c2673d7ababa2). Will this suffice?